### ISSUE TITLE
Copy settings from one object to another - Simple, fundamental.

### ISSUE THREAD

**Issue by PhilBaz** [SoftFever/OrcaSlicer#3576]
**Is your feature request related to a problem? Please describe.**

Yes, currently you can copy a set of object specific settings from one object to another.

**Which printers will be beneficial to this feature**
E.g. Voron/Klipper based printer/etc...

All

**Describe the solution you'd like**

Copy and paste settings. Could be 'copy object settings' in right click menu, along with paste. Or the ability to drag an icon in the object list would be cool.

**Describe alternatives you've considered**

You have to manually change them all. It sucks!

**Additional context**

This seems like a fundamental workflow shortcoming. This is a logical and necessary feature for Orca. :) Hail the Whale!

**Comment by DARKFiB3R** [comment]
This would be so helpful right now.

Also, once you have made changes to the settings of a multi part object, you can no longer Reload from disk or Replace with STL, which sucks.

**Comment by PhilBaz** [comment]
Bumping as this is fundamental functionality that is currently missing.

**Comment by PhilBaz** [comment]
Bumping as this is fundamental functionality that is currently missing.

**Comment by SamuraiBuddha** [comment]
just came across the need for this today, I second OP's suggestion!

**Comment by Wang-Thunder** [comment]
Yup, bump.

**Comment by lukasmalkmus** [comment]
Yes, just wanted to do something like this today, would be very helpful.

**Comment by Thegribbs** [comment]
Bump. this would be an amazing feature to have in the the 'view all objects settings pup-up menu.

**Comment by brysonhyte** [comment]
The desired behavior is achievable by selecting the part you want to clone from first, then selecting the parts you want to clone the settings to.  Once they are all selected, right-click on the main part and select Assemble.  Once the parts are grouped into an assembly, you can ungroup them by right-clicking and selecting Split->To Objects.

**Comment by carlosjln** [comment]
Indeed, it would be quite helpful for us folks who work with repeated settings on different STLs, since there isn't a way to define a process that enables height range modifiers and all settings while just swapping the object.

The menu options could be inserted after "Height Range Modifier" as "Copy Settings" and "Paste Settings",

![image](https://github.com/user-attachments/assets/5f0943ea-75e5-4b4a-9eed-6ab5e9e1eafc)

**Comment by DARKFiB3R** [comment]
Just wanted to say thanks again to [brysonhyte](https://github.com/brysonhyte)

The workaround has been so helpful while making loads of tiny modifications to the model I'm working on, while I try to really dial it in for the print process. I'd usually be pulling my hair out by this point.

I suppose saving my current settings as a profile would help, but then I'm also constantly messing with slicer settings too, and I don't want a ton of profiles cluttering up the list.

**Comment by emutier** [comment]
my use case: temperature calibration on linux. 
the stl is broken so I wanted to I fixed in an online tool. however, there is no way copy the object settings settings from the calibration stl

**Comment by holser** [comment]
Alternatively, there should be a clone mode. In that mode when user edits one object the changes will apply to all "clones". That will help users when they need to adjust object later when all clones are placed on plate.

**Comment by zombodotcom** [comment]
how is this not a thing still

**Comment by Yankeedoody** [comment]
Bump! A "copy object settings" button would be incredibly useful!

**Comment by Andreuha** [comment]
Bump... came here looking for a way to apply settings from one object in a project to another, so sad this is impossible. Or let me save an object's settings as a preset.

**Comment by DARKFiB3R** [comment]
> Bump... came here looking for a way to apply settings from one object in a project to another, so sad this is impossible. Or let me save an object's settings as a preset.

Soon. It's already available in BambuStudio.

**Comment by carlosjln** [comment]
Note stale, just unattended. Feature pretty much needed.

**Comment by 1NEGROUP** [comment]
Bump feature needed

**Comment by LJKraus** [comment]
Copy/Paste object Process settings is already available with a right-click in the [Nightly build for OrcaSlicer 2.3.2-dev.](https://github.com/OrcaSlicer/OrcaSlicer/releases/tag/nightly-builds)

<img width="270" height="177" alt="Image" src="https://github.com/user-attachments/assets/a4c80883-00d1-44c6-ae78-f49adb1640aa" />

**Comment by Felix14-v2** [comment]
Can be closed as completed, I guess?

**Comment by holser** [comment]
Yes, the feature is added to beta and works as expected. Thank you very much!
